corehq/apps/commtrack/processing.py

```diff
@@ -66,6 +67,21 @@ def _create_model_for_stock_report(domain, stock_report_helper):
         domain=domain,
     )
 
+LedgerValues = namedtuple('LedgerValues', ['balance', 'delta'])
+
+
+def compute_ledger_values(lazy_original_balance, report_type, quantity):
```

Comment:
Could be, but I don't really think of it as optional, just sometimes unused, which you wouldn't know beforehand unless you did the same check as the function did. Anyhow, I put it first because it's sort of a chronological order thing "start at the `original_balance`, then based on the `report_type`, update the `quantity`", but that's obviously pretty subjective